Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -670,6 +670,7 @@ bool TryInitialize() {
         std::vector<Frame::Plane> Planes{};
         std::vector<Frame::Mesh> Meshes{};
         std::vector<FeaturePoint> FeaturePointCloud{};
+        std::pair<bool, Frame::Space> EyeTrackerSpace{};
```

Comment:
>EyeTrackerSpace [](http://example.com/codeflow?start=38&length=15)

should this be referenced by the xr frame? right now although this struct is populated it doesn't seem like it would ever be correctly populated for the frame on iOS